src/v2/view-builder/views/webauthn/ChallengeWebauthnView.js

```diff
@@ -75,11 +75,11 @@ const Body = BaseForm.extend({
         });
       }
     });
-    const options = {
+    const challengeData = authenticatorData.contextualData.challengeData;
+    const options = _.extend({}, challengeData, {
       allowCredentials,
-      userVerification: authenticatorData.contextualData.challengeData.userVerification,
```

Comment:
Why are we removing `userVerification` attribute? Did not quite understand this.

Reply:
@srinivasanagandla-okta  the userverificaiton is already in challengeData which we are extending from and we are not overriding it unlike the challenge.

Reply:
So what is the net new change then since challenge attribute seems to be the same? Sorry for my JS ignorance.

Reply:
NP! Basically, before we had an object `options` which we passed into the WebAuthn API. It looked like this:
```
{
  allowCredentials: [...],
  userVerification: "someString", // equal to authenticatorData.contextualData.challengeData.userVerification
  challenge: "challengeNonce" // equal to CryptoUtil.strToBin(authenticatorData.contextualData.challengeData.challenge)
}
```

**The issue**: we needed to pass the WebAuthn-relevant `extensions` object over into the final `options` object. To achieve this, we're using `challengeData`, which already contains that `extensions` object (but was going unused). `challengeData` looks like this:
```
{
  userVerification: "someString",
  challenge: "someNonce",
  extensions: { "appid": "https://org.okta1.com }
}
```

So, we're using the [underscore.js library's extend method](https://underscorejs.org/#extend): `_.extend({}, challengeData, {...});`; it takes an empty object `{}` and extends it by using the entire above `challengeData` object, then the `allowCredentials` + `challenge` values in the third object argument, and sets them into the `{}` object.[1]

So, with this new change, we:
1. Still add `allowCredentials` into the final `options` object
2. There's no need to specify `userVerification` because it's part of the `challengeData`, which is going into `options`
3. We still do still post-process the `challenge` nonce (put it into a binary data encoring format)
4. Similar to `userVerification`, there's no need to specify `extensions` now because it's part of `challengeData`

Let me know if that makes sense!

[1]Since it's copied by source, technically we could omit the `{}` object, but this way we just have consistent code w/ v1: https://github.com/okta/okta-signin-widget/blob/587acb9220137d324ab4d556b9dbf32c9e5cc8cc/src/VerifyWebauthnController.js#L97-L100